CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Fixes bug where context was not being used at all during startVM (#86)
 * Updates the jailer's socket path to point to the unix socket in the jailer's workspace (#86)
 * Fixes bug where default socketpath would always be used when not using jailer (#84).
+* Update for compatibility with Firecracker 0.17.x
```

Comment:
The `# [ Unreleased ]` needs to be renamed to `# v0.17.0`

Reply:
Not until we actually release a 0.17.0 version of the SDK, which is not what we're doing here.